irohad/model/execution/impl/command_executor.cpp

```diff
@@ -223,12 +223,10 @@ namespace iroha {
         const Command &command,
         ametsuchi::WsvQuery &queries,
         const std::string &creator_account_id) {
-      // TODO: no additional checks ?
```

Comment:
Why there's no need in checks?

Reply:
Some additional validation rules can be defined from the requirements